app/script/conversation/ConversationService.coffee

```diff
@@ -73,6 +73,43 @@ class z.conversation.ConversationService
         start: conversation_id
 
   ###
+  Retrieves all the conversations of a user.
+  @param limit [Integer] Number of results to return (default 500, max 500)
+  ###
+  get_all_conversations: (limit = 500) =>
```

Comment:
Why are we doing this on the service level now? It's more logic than belongs here isn't it?